indra/newview/scripts/lua/LLDebugSettings.lua

```diff
@@ -0,0 +1,24 @@
+leap = require 'leap'
+
+function check_response(res)
```

Comment:
Let's make this a `local` function. I like it. Does it work when you pass an unknown group or control name?

Reply:
Yes, I've checked it.